src/librustc_traits/chalk/lowering.rs

```diff
@@ -1,3 +1,36 @@
+//! Contains the logic to lower rustc types into Chalk types
+//!
+//! In many there is a 1:1 relationship between a rustc type and a Chalk type.
```

Comment:
Seems like a word is missing here. Just a guess:

```suggestion
//! In many cases there is a 1:1 relationship between a rustc type and a Chalk type.
```

Reply:
Thanks :)